cmd/kubeadm/test/token_test.go

```diff
@@ -28,12 +28,11 @@ const (
 	TokenExpectedRegex = "^\\S{6}\\:\\S{16}\n$"
 )
 
-var kubeadmPath string
+var kubeadmPath = flag.String("kubeadm-path", filepath.Join(os.Getenv("KUBE_ROOT"), "cluster/kubeadm.sh"), "Location of kubeadm")
```

Comment:
sorry, the global package-level flag registration is the objection, not whether it's in or out of an init() block